test/e2e/common.go

```diff
@@ -60,14 +61,10 @@ func Byf(format string, a ...interface{}) {
 	By(fmt.Sprintf(format, a...))
 }
 
-func Logf(format string, a ...interface{}) {
-	fmt.Fprintf(GinkgoWriter, "INFO: "+format+"\n", a...)
-}
-
 func LogFromFile(logFile string) {
```

Comment:
I am not sure about using BMO e2e `Logf`. `AnnotateBmh` makes sense but I think it's better to keep `Logf` here. This is something that can be different in BMO and CAPM3.

Reply:
Yes, I think that makes sense, but could you elaborate on what could be the differences? This function is greatly basic, I don't see much space for diversion.